browser.ts

```diff
@@ -0,0 +1,39 @@
+import { fromDom } from 'hast-util-from-dom';
+import { type Code, type Parent, type Root } from 'mdast';
```

Comment:
Yes, those are equivalent. Because there are no ESLint rules to force a consistent way to write type imports, I have decided to disallow type imports in my personal ESLint config. However, Playwright doesn’t support unannotated type imports, so I decided to enforce type imports in this project for consistency and use a `type` annotation per import specifier. Using such annotations per specifier allows to combine both type imports and value imports in a single statement.